Jenkinsfile

```
@@ -26,35 +26,36 @@ node('docker&&linux') {
 
     timestamps {
         stage('Generate Plugin Data') {
-            docker.image('maven').inside {
-              sh 'mvn -PgeneratePluginData'
-            }
+          infra.runWithMaven(
+            'mvn --no-transfer-progress -PgeneratePluginData',
+            /*jdk*/ '17',
+            /*extraEnv*/ null,
+            /*addToolEnv*/ false
+          )
         }
 
         /*
          * Running everything within an nginx container to provide the
          * DATA_FILE_URL necessary for the build and execution of the docker
          * container
          */
-        docker.image('nginx:alpine').withRun('-v $PWD/target:/usr/share/nginx/html') { c ->
+        docker.image('nginx:alpine').withRun('-p 80:80 -v $PWD/target:/usr/share/nginx/html') { c ->
 
             /*
              * Building our war file inside a Maven container which links to
              * the nginx container for accessing the DATA_FILE_URL
              */
             stage('Build') {
-                docker.image('maven:3-adoptopenjdk-11').inside("--link ${c.id}:nginx") {
-                    withEnv([
-                        'DATA_FILE_URL=http://nginx/plugins.json.gzip',
-                    ]) {
-                        infra.runWithMaven(
-                            'mvn -Dmaven.test.failure.ignore verify',
-                            /*jdk*/ "8",
-                            /*extraEnv*/ null,
-                            /*addToolEnv*/ false
-                          )
-                    }
-                }
+              withEnv([
+                'DATA_FILE_URL=http://localhost/plugins.json.gzip',
+              ]) {
+                infra.runWithMaven(
+                    'mvn --no-transfer-progress -Dmaven.test.failure.ignore verify --version',
+                    /*jdk*/ '8',
+                    /*extraEnv*/ null,
+                    /*addToolEnv*/ false
+                  )
+              }
```

```suggestion
                infra.runMaven(['-Dmaven.test.failure.ignore',  'verify'])
              }
```

As per https://github.com/jenkins-infra/pipeline-library/blob/789ef58fe3490019c92e8e59296a9c1d2aa96f91/vars/infra.groovy#L163